composer.json

```diff
@@ -14,13 +14,18 @@
         "symfony/framework-bundle": "~2.7|~3.1",
         "symfony/finder": "~2.7|~3.1",
         "symfony/console": "~2.7|~3.1",
-        "symfony/intl": "~2.7|~3.1"
+        "symfony/intl": "~2.7|~3.1",
+        "symfony/asset": "~2.7|~3.1",
```

Comment:
this should not be a mandatory requirement (it may be needed as a dev requirement if the testsuite uses it).
The bundle does not forces you to use the Asset component to load the JS file.